CONTRIBUTING.rst

```diff
@@ -279,6 +279,41 @@ In another shell:
 
 The MLflow Tracking UI will show runs logged in ``./mlruns`` at `<http://localhost:3000>`_.
 
+Testing React Component
++++++++++++++++++++++++++++++++++
+Add a test component in the same directory as the newly created React component. 
+For example, ``CompareRunBox.test.js`` should be added in the same directory as ``CompareRunBox.js``.
+Next, in ``mlflow/server/js``, run the following command to start the test.
+
+.. code-block:: bash
+
+   yarn test
+
+Once the test is loaded, you will be prompted with an option menu as below:
+
+.. code-block:: bash
+
+  No tests found related to files changed since last commit.
+  Press `a` to run all tests, or run Jest with `--watchAll`.
```

Comment:
`yarn test` executes `yarn test:base --watchAll=false` which should run tests in non-interactive mode.

Reply:
On my machine, `yarn test` runs tests in non-interactive mode.

Reply:
That seems strange. On my mac, it ran interactively. I will change the wording to yours.